package.json

```diff
@@ -38,16 +40,7 @@
   "engine": {
     "node": ">=0.8.x"
   },
-  "licenses": [
-    {
-      "type": "GPL",
-      "url": "https://github.com/wikimedia/jquery.i18n/blob/master/GPL-LICENSE"
-    },
-    {
-      "type": "MIT",
-      "url": "https://github.com/wikimedia/jquery.i18n/blob/master/MIT-LICENSE"
-    }
-  ],
+  "license": "(MIT OR GPL-2.0)",
```

Comment:
That's true there isn't a central place to define the licenses, other than this file now. The files I spot checked all say "GNU General Public Licence 2.0 or later" and that's also the license we use for other code.